lib/get/get-manager.js

```diff
@@ -225,7 +225,7 @@ class GetManager {
       return { success: false, result: null, results: null }
     }
 
-    const executorOptions = this.goconfig.executor.getOptions('file')
```

Comment:
Why the change from file to project here?

Reply:
It did not make any sense to me to run `go get` in the context of the currently open file. This could have been any file without understanding it, and even falls back to the 'project' options in case no file is currently open.
I can revert it if necessary.

Reply:
no need to revert, was just curious